SUTrivBot/Models/IGameState.cs

```diff
@@ -0,0 +1,13 @@
+using DSharpPlus.CommandsNext;
+
+namespace SUTrivBot.Models
+{
+    public interface IGameState
+    {
+     
+        public string getGameName();
+        public void play(CommandContext ctx);
+        public void next(CommandContext ctx);
+        public void stop(CommandContext ctx);
```

Comment:
.NET nameing convention is for method names to be capitalized. 
so GetGameName(), Play(...), etc..

Reply:
Sounds good, will update!